docs/CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Add new SOURCE option into FlawReferenceType (OSIDB-1556)
 - Add Celery worker concurrency
 - Add new NVD option into FlawSource
+- Add external_id attribute on Snippet model
```

Comment:
I know that we do not obey it always but once we agreed that the CHANGELOG should contain only user-facing changes. It is probably no issue to put there more, just that we were trying to behave like a service to be consumed and avoid changes which do not tell anything to the consumers (those would only go to commit messages). Adding **technical** GitHub label then turns of the CHANGELOG checking pipeline.